src/Runner.Listener/Configuration/ConfigurationManager.cs

```diff
@@ -552,5 +553,11 @@ private async Task<GitHubAuthResult> GetTenantCredential(string githubUrl, strin
                 }
             }
         }
+
+        private static class RunnerCredentialEvents
```

Comment:
move this to `Runner.Common.Constants.cs` and just name it `RunnerEvent.Register` and `RunnerEvent.Remove`

Reply:
Moved to constants